content/docs/gto/user-guide/index.md

```diff
@@ -1,122 +1,220 @@
 # User Guide
 
-GTO helps you build an Artifact Registry out of your Git repository. It creates
+GTO lets you build an Artifact Registry out of your Git repository by creating
 annotated [Git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) with a
-[special format](#git-tags-format), and manages an `artifacts.yaml` file.
+[special format](#git-tags-format).
 
-<admon type="tip">
+## Finding the right artifact version
 
-Storing large files in Git repos is not a good practice. Avoid committing your
-ML artifacts to Git. You can [use DVC](/doc/gto/user-guide/dvc), Git LFS, or any
-other method to commit pointers to the data, models, etc. instead.
+You may need to get a specific artifact version to a certain environment, most
+likely the latest one or the one currently assigned to the stage. Use `gto show`
+to find the [Git reference] (tag) you need.
 
-</admon>
+[git reference]: https://git-scm.com/book/en/v2/Git-Internals-Git-References
 
-## Annotations in artifacts.yaml
+Get the git tag for the latest version:
 
-Using Git tags to register artifact versions and assign stages is handy, but the
-Git tag itself doesn't contain a path to the artifact files, their type (`model`
-or `dataset`), or any other useful information about them. For simple projects
-(e.g. a single artifact) we can assume the details whenever we consume the
-artifacts (e.g. [in CI/CD](#acting-in-ci-cd)). But for more advanced cases, we
-should codify them in the registry itself.
-
-To keep this metadata, GTO uses a human-readable `artifacts.yaml` file. The
-`gto describe`, `gto annotate`, and `gto remove` commands are used to display
-and manage it's contents.
+```cli
+$ gto show churn@latest --ref
+churn@v3.1.1
+```
 
-<admon type="tip">
+Get the git tag for the version in `prod` stage:
 
-An example `artifacts.yaml` can be found
-[in the `example-gto` repo](https://github.com/iterative/example-gto).
+```cli
+$ gto show churn#prod --ref
+churn@v3.0.0
+```
```

Comment:
Feel free to reword this, but I thought it would be helpful to still mention DVC here as a way to get the model. We can also include the `dvc get` command example here and summarize the other integration points with DVC, and then drop the section below about `artifacts.yaml` so we don't have a whole section of the guide that's not really about GTO.

````suggestion
```

GTO doesn't provide a way to deliver the artifacts, but you can use DVC 
or any method to retrieve files from the repo. With DVC, you can use [`dvc get`]:

```cli
$ dvc get $REPO $ARTIFACT_PATH --rev $REVISION -o $OUTPUT_PATH
```

<admon>

You can also use DVC with GTO to:

* [Store large artifacts] (models and data) and track pointers to them in your repo.
* [Keep artifact metadata] like the path or type (`model` or `dataset`). To see an example, check out the [`example-gto` repo].

</admon>

[dvc get]: https://dvc.org/doc/command-reference/get
[store large artifacts]: https://dvc.org/doc/start/data-management/data-versioning
[keep artifact metadata]: https://dvc.org/doc/user-guide/project-structure/dvcyaml-files#artifacts
[example-gto repo]: https://github.com/iterative/example-gto/blob/main/dvc.yaml
````